tslint.json

```diff
@@ -21,10 +21,6 @@
       "avoid-escape",
       "jsx-double"
     ],
-    "no-unused-variable": [
```

Comment:
ahh I see. Okay then, it would be nice to find a substitute for this rule in future...